### PR TITLE
feat: sending a message to eigen on successful order

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **Feat**

This PR is part of [MOPLAT-311]

### Description
We need to get a callback so that we know the order was successful. That way we can do some work after that.

https://user-images.githubusercontent.com/100233/187646529-d92b7988-ab9a-4847-b668-ad7f6c3650ac.mov

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MOPLAT-311]: https://artsyproduct.atlassian.net/browse/MOPLAT-311?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Order/Routes/Review/index.tsx [hunk 1]
```diff
@@ -129,24 +129,28 @@ export const ReviewRoute: FC<ReviewProps> = props => {
...
+            )
+            // We cannot expect Eigen to respond all the time to messages sent from the webview
+            // a default fallback page is safer for old/broken Eigen versions
+            setTimeout(() => {
+              router.push(`/orders/${order.internalID}/status`)
+            }, 500)
+            return
+          }
+
+          if (order.mode === "BUY") {
```
Comment (araujobarret) [inline src/Apps/Order/Routes/Review/index.tsx:149]: I'm from mobile phone here(ooo for the next 2 weeks) but I would recommend a fallback for older Eigen versions that have no interaction with this new dispatched message. Maybe moving line 143 block down?

Comment (pvinis) [reply src/Apps/Order/Routes/Review/index.tsx:149]: what will happen if its an older version eigen? 🤔 the message will be posted, but no interaction will happen, right?

if so, that is totally fine for us.

Comment (araujobarret) [reply src/Apps/Order/Routes/Review/index.tsx:149]: Yep, if that's the expected it's fine!

**Comment by damassi** [comment]
👍

**Pull request merged**